docker.local/build.miner.dev/Dockerfile.integration_tests

```diff
@@ -0,0 +1,37 @@
+# Compile the miner in an un-tagged image so the final, tagged image can be smaller:
```

Comment:
Files `Dockerfile.integration_tests`, `b0docker-compose.yml`, `docker-compose.yml`, `nongenesis_docker_compose.yml` are copies of files under another paths, without any changes. Copy-paste is bad practice because when you will need to fix something, you will need to fix it in several places and this is easy to forget.